xml/thesis_form_ISLANDORA1350.xml

```diff
@@ -701,7 +662,7 @@
               <access>TRUE</access>
               <collapsed>FALSE</collapsed>
               <collapsible>FALSE</collapsible>
-              <description>The year this thesis was issued.</description>
+              <description>The year this book was issued.</description>
```

Comment:
why revert this?

Reply:
I hit the "revert" button on the PR... since this change was in the PR, it got included.

Reply:
ok, though we could have kept this tiny improvement. next time!